requirements.txt

```diff
@@ -1,25 +1,57 @@
-flask
-geoip2
-geopy
-requests
-gevent
-dnspython
-pylint
-cyrtranslit
-astral
-timezonefinder==2.1.2
-pyte
-python-dateutil
-diagram
-pyjq
-scipy
-numpy
-pillow
-babel
-pylru
-pysocks
-supervisor
-numba
-emoji
-grapheme
+aiohttp==3.6.2
+astral==2.2
+astroid==2.4.2
+async-timeout==3.0.1
+attrs==19.3.0
+Babel==2.8.0
+certifi==2020.6.20
```

Comment:
Ah, so a fun dilema here -- you've saved /all/ of the python packages on your machine here, instead of just the project deps. Easiest way to accomplish what you want -- pip3 freeze and then grep for the original package nam, copy and paste. You shouldn't end up with any more requirements here than we started with

Reply:
ok, so I was able to get the requirements specific to the projects using grep and updated the requirements.txt 👍